doc/sql.extensions/README.schemas.md

```diff
@@ -0,0 +1,651 @@
+before `isc_tpb_shared`, `isc_tpb_protected` or `isc_tpb_exclusive`. The format of `isc_tpb_lock_table_schema` is 
+a single byte length followed by the schema name.
+
+## Array support
+
+### `isc_sdl_schema`
+
+When working with arrays using SDL (Slice Description Language), the `isc_sdl_schema` parameter can now be used to 
+explicitly qualify the schema. Its format is equivalent to `isc_sdl_relation`.
+
+## Utilities
+
+### isql
+
+#### Option `-(SE)ARCH_PATH`
+
+This option enables ISQL to pass the search path argument as `isc_dpb_search_path` with every established attachment.
+
+```
+isql -search_path x,y test.fdb
+select RDB$GET_CONTEXT('SYSTEM', 'SEARCH_PATH') from system.rdb$database;
+-- Result: "X", "Y"
+set search_path to y;
+select RDB$GET_CONTEXT('SYSTEM', 'SEARCH_PATH') from system.rdb$database;
+-- Result: "Y"
+
+connect 't2.fdb';
+select RDB$GET_CONTEXT('SYSTEM', 'SEARCH_PATH') from system.rdb$database;
+-- Result: "X", "Y"
+```
+
+```
+isql -search_path '"x", "y"' test.fdb
+select RDB$GET_CONTEXT('SYSTEM', 'SEARCH_PATH') from system.rdb$database;
+-- Result: "x", "y"
+```
+
+Special names with double quotes require different handling depending on the operating system and, specifically, the 
+shell interpreter. On Linux with `bash`, double quotes must be escaped using the backslash (`\`) character. On Windows 
+with `cmd`, double quotes need to be duplicated. The example below demonstrates how to use the same search path on both 
+platforms.
+
+Linux / bash:
+```
+isql -search_path "x, \"y\", z, \"q a\", system" test.fdb
+
+select rdb$get_context('SYSTEM', 'SEARCH_PATH') from system.rdb$database;
+-- Result: "X", "y", "Z", "q a", "SYSTEM"
+```
+
+Windows / cmd:
+```
+isql -search_path "x, ""y"", z, ""q a"", system" test.fdb
+
+select rdb$get_context('SYSTEM', 'SEARCH_PATH') from system.rdb$database;
+-- Result: "X", "y", "Z", "q a", "SYSTEM"
+```
+
+### gbak
+
+To use databases created in earlier Firebird versions with Firebird 6, you must restore a backup using the Firebird 6 
+`gbak` utility. In the restored database, all user objects will be placed in the `PUBLIC` schema.
+
+Firebird includes several built-in plugins that create database objects. Prior to Firebird 6, schemas did not exist, 
+so these objects were only distinguished by their `PLG$` prefix. When restoring a pre-Firebird 6 backup, `gbak` 
+automatically attempts to recreate these objects (and their data) in specific schemas — `PLG$PROFILER`, `PLG$SRP`, and 
+`PLG$LEGACY_SEC` — while removing those located in the `PUBLIC` schema. This process may fail if user-defined objects 
+reference these plugin objects. In that case, `gbak` will display a warning and halt the plugin schema migration 
+process.
+
+#### Options
+
+`gbak` now includes the `-INCLUDE_SCHEMA_D(ATA)` and `-SKIP_SCHEMA_D(ATA)` switches, which complement the previously
+existing `-INCLUDE_SCHEMA` and `-SKIP_SCHEMA` options. If `-INCLUDE_SCHEMA_D(ATA)` is not specified, all schemas are
```

Comment:
Previously existing `-SKIP_D(ATA)` and `-INCLUDE(_DATA)` ?

Reply:
Will fix.